tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py

```diff
@@ -44,11 +74,9 @@ def set_up_dirs_and_db():
     )
 
 
-def test_scene_move(set_up_dirs_and_db):
+def test_ard_reprocessed_l1s(set_up_dirs_and_db):
```

Comment:
should we rename this test to have the word, 'move' inline with the comment below (ie. moving towards self documenting code)?

Reply:
I'll fix the comment